### PR TITLE
[Wasm GC] Fix the order of operands in array.new and struct.new

### PR DESCRIPTION
**Pull request by kripken**

### CODE REVIEW

File: test/let.txt [hunk 1]
```diff
@@ -0,0 +1,42 @@
+;; source code for let.wasm, which was created by wasp
```
Comment (tlively) [inline test/let.txt:1]: Is this file just a drive-by fix because it was previously missing?

**Review by tlively — APPROVED** [review]

File: test/let.txt (hunk 1 above)
Comment (kripken) [reply test/let.txt:1]: Yes. (It was useful to debug this.)
